ProjectManagmentSystem/src/Modules/TasksModule/components/TasksList/TasksList.tsx

```diff
@@ -1,7 +1,135 @@
-import React from 'react'
+import React, { useContext, useEffect, useState } from 'react'
+import Styles from "./TasksList.module.css"
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { AuthContext } from '../../../Context/AuthContext';
+import NoData from '../../../SharedModule/components/NoData/NoData';
 
 export default function TasksList() {
+  const { requestHeaders, baseUrl }: any = useContext(AuthContext);
+  const [tasksList, setTasksList] = useState([]);
+
+  const navigate = useNavigate();
+
+  ////API's
+
+//Get ALL Projects API
+
+const getTasksList = async () => {
+  try {
+    let response = await axios.get(`${baseUrl}/Task/manager`, {
+      headers: requestHeaders,
+    });
+    setTasksList(response.data.data);
+    console.log(response.data.data);
+  } catch (error) {
+    console.log(error);
+  }
+};
+useEffect(() => {
+  getTasksList();
+}, []);
+
+const navigateToAddTask = () => {
+  navigate("/dashboard/tasksdata");
+};
   return (
-    <div>TasksList</div>
+    <>
+      <div className="w-100 compTitle d-flex justify-content-between my-5 bg-white p-4">
+        <h2>Tasks</h2>
+        <div>
+          <button
```

Comment:
What about making this Link instead? The only purpose of this button is navigation , and that is what a Link is for.